Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -1976,6 +1987,9 @@ class RenderStyle {
     static ScrollSnapAlign initialScrollSnapAlign();
     static ScrollSnapStop initialScrollSnapStop();
 
+    static Vector<ScrollAxis> initialScrollTimelineAxis() { return { }; }
```

Comment:
```suggestion
    static Vector<ScrollAxis> initialScrollTimelineAxis() { return { ScrollAxis::Block }; }
```

I think this was correct fwiw.

Reply:
`{ }` is also correct fwiw, just depends on how you handle serialization of `{ }`. As long as it serializes to `CSSValueBlock` that's fine.

Reply:
Yeah, this is handled correctly as it stands.